sdk/core/azure-core/azure/core/pipeline/policies/_retry.py

```diff
@@ -118,7 +118,7 @@ def configure_retries(self, options: Dict[str, Any]) -> Dict[str, Any]:
             "read": options.pop("retry_read", self.read_retries),
             "status": options.pop("retry_status", self.status_retries),
             "backoff": options.pop("retry_backoff_factor", self.backoff_factor),
-            "max_backoff": options.pop("retry_backoff_max", self.BACKOFF_MAX),
+            "max_backoff": options.pop("retry_backoff_max", self.backoff_max),
```

Comment:
can we add some tests via mocking for this?

Reply:
Added some tests.